stan/math/opencl/rev/logit.hpp

```diff
@@ -20,7 +20,8 @@ template <typename T,
 inline var_value<matrix_cl<double>> logit(const var_value<T>& A) {
   return make_callback_var(
       logit(A.val()), [A](vari_value<matrix_cl<double>>& res) mutable {
-        A.adj() += elt_divide(res.adj(), A.val() * LOG_TEN);
+        A.adj() += elt_multiply(res.adj(),
+                                elt_divide(1.0, A.val() - square(A.val())));
```

Comment:
@t4c1 fyi I think this was wrong before? I wrote it to match up to the rev version

Reply:
It seems so. How did the tests pass?

Anyway you can simplify this into:
```A.adj() += elt_divide(res.adj(), A.val() - square(A.val()));```

Reply:
I reverted this change and the tests failed for me locally. It looks like the right tests are running in CI. I don't know. This seems like a canary worth investigating, but if nobody can reproduce it then nobody can reproduce it :/.

Reply:
Oh also I switched to the simplified `A.adj() += elt_divide(res.adj(), A.val() - square(A.val()));`